src/com/amazon/corretto/crypto/provider/ConcatenationKdfSpec.java

```diff
@@ -0,0 +1,71 @@
+// Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+package com.amazon.corretto.crypto.provider;
+
+import java.security.spec.KeySpec;
+import java.util.Objects;
+
+/**
+ * Represents the inputs to ConcatenationKdf algorithms.
+ *
+ * <p>If info or salt is not provided, empty byte arrays are used.
+ *
+ * <p>The algorithmName is the name of algorithm used to create SecretKeySpec.
```

Comment:
```suggestion
 * <p>If info or salt is not provided, empty byte arrays are used.</p>
 *
 * <p>The algorithmName is the name of algorithm used to create SecretKeySpec.</p>
```

Reply:
Turns out this is unnecessary in a java docstring, and the linter will even remove the close tags. 🤦